project/AkkaBuild.scala

```diff
@@ -1382,14 +1385,37 @@ object AkkaBuild extends Build {
 
     val cluster = exports(Seq("akka.cluster.*"), imports = Seq(protobufImport()))
 
-    val parsing = exports(Seq("akka.parboiled2.*", "akka.shapeless.*"))
+    val parsing = exports(Seq("akka.parboiled2.*", "akka.shapeless.*"),
+        imports = Seq(optionalResolution("scala.quasiquotes")))
 
-    val httpCore = Nil // FIXME #15689
+    val httpCore = exports(Seq("akka.http.*"),
+        imports = Seq(streamAndHttpImport("akka.stream.*"),
+            streamAndHttpImport("akka.parboiled2.*"),
+            streamAndHttpImport("akka.shapeless.*")))
 
-    val http = Nil // FIXME #15689
+    val http = exports(Seq("akka.http.impl.server",
+        "akka.http.scaladsl.server.*", "akka.http.javadsl.server.*",
+        "akka.http.scaladsl.client", "akka.http.scaladsl.coding", "akka.http.scaladsl.common",
+        "akka.http.scaladsl.marshalling", "akka.http.scaladsl.unmarshalling"),
+        imports = Seq(streamAndHttpImport("akka.stream.*"),
+            streamAndHttpImport("akka.http.*"),
+            streamAndHttpImport("akka.parboiled2.*")))
 
     val httpTestKit = exports(Seq("akka.http.scaladsl.testkit.*"))
```

Comment:
Yes, it's the same style of testkit that we provide for actors - a set of utilities to make testing simpler.